scripts/population_template

```diff
@@ -313,53 +314,52 @@ else:
                 output)
     # translate input images to centre of mass without interpolation
     runCommand('mrtransform ' + abspath(i.directory, i.filename) +
-                ' -linear linear_transforms_initial/' + i.prefix + '.txt' +
+                ' -linear ' + os.path.join('linear_transforms_initial', i.prefix + '.txt') +
                 datatype +
-                ' input_transformed/' + i.prefix + '_translated.mif')
+                ' ' + os.path.join('input_transformed', i.prefix + '_translated.mif'))
     if useMasks:
       runCommand('mrtransform ' + abspath(i.mask_directory, i.mask_filename) +
-             ' -linear linear_transforms_initial/' + i.prefix + '.txt' +
+             ' -linear ' + os.path.join('linear_transforms_initial', i.prefix + '.txt') +
              datatype +
-             ' masks_transformed/' + i.prefix + '_translated.mif')
+             ' ' + os.path.join('masks_transformed', i.prefix + '_translated.mif'))
   # update average space to new extent
-  runCommand('mraverageheader ' + ' '.join(['input_transformed/' + i.prefix + '_translated.mif' for i in input]) + ' average_header_tight.mif')
+  runCommand('mraverageheader ' + ' '.join([os.path.join('input_transformed', i.prefix + '_translated.mif') for i in input]) + ' average_header_tight.mif')
   runCommand('mrpad -uniform 10 average_header_tight.mif average_header.mif -force')
-  runCommand('rm -f average_header_tight.mif')
+  delFile('average_header_tight.mif')
   if useMasks:
     # reslice masks
     for i in input:
       runCommand('mrtransform ' +
-               ' masks_transformed/' + i.prefix + '_translated.mif' +
-               ' masks_transformed/' + i.prefix + '.mif' +
+               ' ' + os.path.join('masks_transformed', i.prefix + '_translated.mif') +
+               ' ' + os.path.join('masks_transformed', i.prefix + '.mif') +
                ' -interp nearest -template average_header.mif' +
                datatype)
     # crop average space to extent defined by translated masks
     mask_filenames = []
     for i in input:
-      mask_filenames.append('masks_transformed/' + i.prefix + '.mif')
+      mask_filenames.append(os.path.join('masks_transformed', i.prefix + '.mif'))
     runCommand('mrmath ' + ' '.join(mask_filenames) + ' max mask_translated.mif' )
-    runCommand('rm -f mask_translated_smooth.mif')
     runCommand('mrcrop ' + 'average_header.mif -mask mask_translated.mif average_header_cropped.mif')
     # pad average space to allow for deviation from centre of mass alignment
     runCommand('mrpad -uniform 10 average_header_cropped.mif -force average_header.mif')
-    runCommand('rm -f average_header_cropped.mif')
+    delFile('average_header_cropped.mif')
```

Comment:
Currently `delFile` does not check for the `-continue` option. Also, we would need to implement the `-continue` check for shutil's `copy2`, `move`, `rmtree` and `copytree`. I think it would be best to use the generic `runFunction` that takes any python function to do all of this. 

We could still have the same convenience functions:

    def delFile(path):
        runFunction(os.remove, path)

Currently, the output of that is:

`Function: posix.remove(linear_transforms_4/01.txtdecomp)`